scripts/EN/Harry

```diff
@@ -7983,13 +7983,13 @@ CleanScreen(); // 0xa250: 00000064
 LoadBackground(0x80); // 0xa254: 00001069
 DrawBackground(); // 0xa25a: 00000065
 PlaySoundFx(0x39, 0x1); // 0xa25e: 00002072
-ShowText((3616)"¥"Kaufmann...¥" Kaufmann has shrunk into the corner of the hall, as if he was driven there. One of the walls is covered with a disgusting spatter, and he is already dead on the floor."); // 0xa266: 0000100c
+ShowText((3616)"¥"W4Kaufmann…¥"¥nW3Kaufmann has shrunk into the corner of the hall, W1as if he was driven there. ¥nW2One of the walls is covered with a disgusting spatter, W1and he is already dead on the floor."); // 0xa266: 0000100c
 HandleInput(); // 0xa26c: 00000010
-ShowText((3617)"Upon seeing his injury I sank to the ground. I sat with my legs crossed on the hard tile floor, at a loss as to what to do. That girl is walking around and destroying everything she sees with the gun."); // 0xa270: 0000100c
+ShowText((3617)"I sank to the floor in that very spot.¥nW2I sat with my legs crossed on the hard tile floor, W1at a loss as to what to do.¥nW2That girl is walking around and destroying everything she sees with the gun."); // 0xa270: 0000100c
```

Comment:
Changed first sentence to remove the error and make it match the Japanese finally.

(I mistyped injury in Japanese, but it was spot.)